backend/src/db/index.js

```diff
@@ -7,7 +7,7 @@ export let dbInstance = undefined;
 const connectDB = async () => {
   try {
     const connectionInstance = await mongoose.connect(
-      `${process.env.MONGODB_URI}/${DB_NAME}`
```

Comment:
don't remove this, let it be

as, DB_NAME is coming from /backend/src/constants.js